benefits/enrollment/views.py

```diff
@@ -179,7 +179,7 @@ def success(request):
 
     page = viewmodels.Page(title=_("enrollment.pages.success.title"), headline=_("enrollment.pages.success.headline"))
 
-    if verifier.is_auth_required and session.logged_in(request):
+    if verifier.is_auth_required and session.logged_in(request) and verifier.supports_sign_out:
```

Comment:
Since `verifier.supports_sign_out` already checks `verifier.is_auth_required`:

```suggestion
    if verifier.supports_sign_out and session.logged_in(request):
```

Reply:
Thanks! Done in https://github.com/cal-itp/benefits/pull/1462/commits/3fc8ee11708d80beb8e1de32774c4ac07c9ff84b